intranet/apps/emailfwd/tests.py

```diff
@@ -31,3 +31,12 @@ def test_email_fwd(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(1, SeniorEmailForward.objects.filter(user=user, email="nonexistent@tjhsst.edu").count())
         self.assertIn("Error adding forwarding address.", list(map(str, list(response.context["messages"]))))
+
+
+class GetSeniorForwardsTest(IonTestCase):
+    def test_getting_forwards(self):
+        """Test ability to retrieve senior forwards."""
+        user = get_user_model().objects.get_or_create(username="awilliam", graduation_year=get_senior_graduation_year(), user_type="student")[0]
+        self.login()
+        # need to figure out how to read the data, which goes to stdout
```

Comment:
It might be easiest to log this information in the view and then use tests to check for the logs instead of checking stdout. The other option is to have this command save to a file (which might actually be better for our use case) and then it's fairly easy to test the file output.

Reply:
Doing some Googling, I came across [`call_command`](https://docs.djangoproject.com/en/2.2/ref/django-admin/#running-management-commands-from-your-code), which seems to be useful for this stuff (and it has `stdout` redirection!) I'm not sure if it's the cleanest solution, but I suppose it's worth a try.

Reply:
That sounds good.